Sources/Fluent/Query/Log.swift

```diff
@@ -14,7 +14,7 @@ public struct Log {
     }
 
     /// Create a log from raw sql and values.
-    init(sql: String, values: [Node]) {
+    init(sql: String, values: [Node] = []) {
```

Comment:
Given some of yesterday's security questions, I'm wondering if a quick comment here and not using default values might help